include/grpc/impl/channel_arg_names.h

```diff
@@ -402,6 +402,11 @@
 #define GRPC_ARG_EXPERIMENTAL_STATS_PLUGINS "grpc.experimental.stats_plugins"
 /** If non-zero, allow security frames to be sent and received. */
 #define GRPC_ARG_SECURITY_FRAME_ALLOWED "grpc.security_frame_allowed"
+/** A comma separated list of supported transport protocols. If non-empty, allow
```

Comment:
Can we go into a bit more detail about:
(a) How the list differs between the client and server side?
(b) What are the guarantees about the negotiation? E.g. not distinguishing between the ALTS vs. SSL implementations, but more about what guarantees does gRPC I have if I set this, if a protocol is actually negotiated, etc.?

Reply:
Right now it's not implemented at all, so I don't think we have any details to provide
I have that in the `NOTE` a few lines down

We could put in likely implementation details, but I'd rather just detail out this comment once the impl is finalized?

Reply:
Fair point - can we add a TODO to come back and add these details once they are finalized?

Reply:
Done